docs/testing-ports.md

```diff
@@ -4,7 +4,7 @@ simultaneously, so port conflicts will fail the tests.  Make sure
 to update this manifest when adding a test that starts a server.
 
 ## generator-react-server 
-3010, 3011
+3000, 3001
```

Comment:
Nope, you're right. 😞 
![image](https://cloud.githubusercontent.com/assets/115908/17347489/065f22cc-58c5-11e6-95ef-347bd9e0a3e2.png)